docker/run-prod.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -xe
 ./docker/run-common.sh
-gunicorn wsgi.app:application -b 0.0.0.0:${PORT:-8000} -w 2 --error-logfile - --access-logfile - --log-level ${LOGLEVEL:-info}
+gunicorn wsgi.app:application -b 0.0.0.0:${PORT:-8000} -w ${GUNICORN_WORKERS:-2} --error-logfile - --access-logfile - --log-level ${LOGLEVEL:-info}
```

Comment:
Only note is that it might be better to make the variable non-server-specific. Maybe `WSGI_WORKERS`? On basket it's `WSGI_NUM_WORKERS`, but that seems verbose now. It's fine if you like `GUNICORN_WORKERS` too, just a suggestion.

r+